### PR TITLE
fix(922110): remove possesive quantifiers

### LINKED ISSUE #2988: Forbidden possesive quantifiers in rule 922110
Rule 922110 has possessive quantifiers that prevent it to work with other regexp engines like re2.

Here we are using them twice: https://github.com/coreruleset/coreruleset/blob/v4.0/dev/rules/REQUEST-922-MULTIPART-ATTACK.conf#L51 and https://github.com/coreruleset/coreruleset/blob/v4.0/dev/rules/REQUEST-922-MULTIPART-ATTACK.conf#L69

### Expected behavior

We need compatibility with other engines.

### PR DESCRIPTION
**Pull request by fzipi**
Signed-off-by: Felipe Zipitria <felipe.zipitria@owasp.org>

Fixes #2988.

### CODE REVIEW

**Review by theseion — COMMENTED** [review]
LGTM, but make sure to rebase onto https://github.com/coreruleset/coreruleset/pull/2984

**Comment by fzipi** [comment]
Well, now that the inclusion is working, I moved the original 920600 and now both files are including the new one.

File: util/regexp-assemble/data/include/charset-specification.data [hunk 1]
```diff
@@ -0,0 +1,107 @@
+##! The expression generated from this file matches a full HTTP
+##! `Accept` header by following the specification as far as
+##! necessary, while restricting the `charset` parameter
+##! to a list of explicitly allowed values.
+##! Where possible, the expression matches tokens "loosely",
+##! to reduce complexity and the risk of false positives.
+
+##! As this specification might be use in more rules, we now
```
Comment (theseion) [inline util/regexp-assemble/data/include/charset-specification.data:8]:
```suggestion
##! As this specification is used by several rules, we now
```

**Review by theseion — CHANGES_REQUESTED** [review]
Awesome!!

**Review by theseion — APPROVED** [review]

**Pull request merged**